test/core/modules/shadow.js

```diff
@@ -0,0 +1,4 @@
HDu2KUwCIVNAAK111IIyGAgJ25QTMEhAEAHGCLAhDRIQQaiRJaoEaoSMDRGBbcCIYlvO7OcztQ+oApsntpEHoUZvwXASBOkiPVBA60GVMUIWYGSwCwkwILBBCqJ1snaeCAhM6+ZwOXF4fkTNRA9yu2Cc32M/J7UWOVbW8yBH4j1pNuIPbJAAQWWhLro6OTZgYBcGRKOAJpDNk8wiQtjF/nhi7AvORCRPWkwIEa2BRBWUDS4MrOuJrkajI8NaJ/YcIGPMtZ4zXV3T+0xMBQ5m/n87ASXKO7kXZeMqhFjZGTYXFxOWMAEI24DorbPmIBQ4oEUyclBloOgKylD8wbYPYksOPRhVLHvS+SNrfYbWQUERLIg7OtunH3L90TOux4laH9D2SC5vqPVEbidYV9q2UvdvyW9eom9f4XWl7YULXAU2hRkUUsMtcA9q7sSbtzzeP3D52afUZ5/y7MW/of7NFW9v/5Gv3vwf1C/uuXouLo4HWnSqhKt40kNkGmEUQJhRg8pkCpgltnWw3p/Zl4XcElm4kqpE5p0JQgwn0ySmywZ9oLbgaEz9GmlmGztZg2QiPbNX4/78yL/80z/y+y9+w1h3nMZpQFQlLsCACgPiHRtswsJA2ahMa+CCYQgJCUrGQLRGYgITiMJQBgQyyDxRBNGDEESIsJkE1xcHDnNDJG2amPpMhJCKJGHZmB8f+Jji/R/9KT/54Z/wX377Jf/7L37F7+4eWd1AjYERYEO6UAQQGJACEFCAaLxjYYwURDQcgQEJqpLKpBuEKCAIsHEN7EFQiOBJCCTRBa2Sljszjcvc+JiN9ww3ucD+mrG9RG1hPYhXzeQhON4849d39/zreQU1ClECIiAEIZAoGyOaGhGd6kHuSWuNiuBhNffnlf3ZBc1BtYnUgUZRA1LJNF3jlqz1wFje0A9XaJ5pz2+IsbKdH4iLmfnqgvGwUnuyLTuuRpUQQWEo8URhqhLxzjSRafaxM1XHmGhBmzvCjH1FuYI3XAsCPHY8NmRwif208/j1W179/Dfc/+Yrxu09ddpw7jhN9CAV6OJIe+8FeXHN3bbzxe9e8ouf/5avv7ol1yRoiEAIJ+QYVCUuY6BSpIwEFGSKJyWogugBBOXCgAlcRfRGFmQmZUAmCMpBKTAQiLLBCRJl0yNoITKLsEjvVBa5Dxqi9oEzgcJAM6gS7xupoKKRFsKMbcHbRqtBF8ytcWBwUxs/vXnBzz7/jJ9872N+/NEnPGvXjEVUa5SS2ld0DCKC1jqId4o4dLpuqPUD8tlzxjczcZhpx5naFzJ3AohogAgF5SQq2d++ZP39L5g+/IRogQfvTLTDgcN85ObTTzn4keXNHVpPaF3QuuDzwjgt1J7UnqhADsJgCghsM8YgJBSNACoL1hUkLIgQIeFMvhOACzWBoSiQGXPA8wOH73/E8aNLtocv8LoyjZ1KyF1IFygbtZwZ54WxruzLjtpEU6MZxrLgsRMRuAZP+uUVisbj7WtOy+D1Mvjlm3tejST7jN2gBXIRCeXCQFGIQDIoeFJVRDSMsM1YN5bHR8a6MN1AtEBpXAOiAUbiO8JIRiqkoFL8QYBB5p1CCCEKCAlFI0dhB6gRAtkoRNk86SEqk9o2attxJs0mbIRpEqFCOQgXLUQpwEJNVBXiiXBBiHcMreEIMncsg40k1BqhjkZQLgjT5pl+mGhzQz0gTDtOHK8vyXOy3O2s+yP5OMhtozJRAS6kBgHiHQER9NYoASPBiUkKEwoCkTwRAiRTFKQYuZGZhEVDFBDqQEMSEYEiMEVYIKja8Z5UgC1iBOzgWnntb4kQkrik0y+OTHNnrUQ0Ws20MmMkrTdaDRLITIwxQSohgjFM7yZ6EWogkIAQRlhBOUGitQCSzJ0WAhpIDIrzssF55dg75ghtofNHZjWIjiVQkApOmvjt3rhY4eP5kuPVxPHmmub3uNCOaqWNZK6B727ZfvVrRhZ+84BOC5SoMmPsrPvOuid2YQk1EdOOl4314ZHxcMc4b1z/yQ95/5Pv8d4P3+Prr6+5fXzJ/fnAkSOiERJqMHJgEnsAA3nQunAH1wZjh20lT2fytMBIch+ojKuwTbSgZDIHdDNfddpF4L5jnXE18IHCjBLJxDY6yy6Gg6++/oZf/vxfOb25g5FgsI0rwcYGY5CRBAZFQBnzjo0okBHCmCfGhAJjRNBUGJMUDQgFtpGEQsgmIkBGgtYaTyLgYmp88sELri+O7AIisKHKzIeZJrOPjXF/zzw3pvbIjz74iO//8D/y+U9+yv/63/6ef/zlryALCbKgAJUgZrLANmVA4OL/JdQCBEIogiBwCFdRY8djhyqkIEJgYychkJJw0tSIMoexMRkmissGB+BFiI+PM9+LM1fLW6b7V+S3XzLdv+LAxjY3Hj9+Hxva1TOmLL66f8O5X7KrERG0PqFoEB0DBkLC0XCbUINWok2BKE774NX9mc+/9xFMR7ZhHAeYgz51xjhTudBvjuxbkusbYjxD8zPcRFwciYsju+6IHkzHA0qgTpwfB6MK845BgG0wYGFDZZEyscPYE4YJRNUgZOiACtjAK67CuYMTubEtxevffMnv/u9f8fDr3+E3D/S9aGlqFH2aGNHwYSKe3VBX19wvg1/+5nf8y9//Cy+//JbxuBFMEEFEwzakyCFc4oklhpNCBAERZCW2QWAFuQ8UgRFEUJVEa/TWSaBUVBURYEAtmKbOUNK6aE1EF6JTghbBk7KxwWkocBkroKBcBKbJKHjHOAe1LewqXIkQNTY0NuSkCybDi9z58xfP+V9+9Dk/+egFh31l/PrXbPNzpo9+QDsULkGfCII+H8hmFMISCJgnpg8+pK8/4Pz7X8Ptay4PE2PZyPMOBjVRWdTYwUlEkKcHtpdfcnh2w3z9nK025MY8z1wcOq2ZvLuF5Y7YTuxvbuHhnhgDpVGBs3BBk5BEVRGt0VonxyARYWGbGhumITWiN9SEbLBBwi4kES3IsTMkPEH/5D0uf/AJ8cF7
```

Comment:
Change the options input to something other than default and update the benchmark accordingly!

Reply:
From where should I get the benchmark? There are lot of websites available to convert img to datauri. So I get different datauri's. How do I match it with my output image?

Reply:
Just use your module to generate the image in IS then open the image in a new tab or you can use the sequencer object to get the data uri, sequencer.steps holds the array of all the steps you just go through it you'll find it easily! Probably in `sequencer.steps[1].output.src`

Reply:
Done @blurry-x-face 
![Screenshot from 2020-01-26 11-40-30](https://user-images.githubusercontent.com/32176296/73131426-cd962b00-4030-11ea-8f2a-19bc6555af5f.png)